test/locust-users.py

```diff
@@ -22,10 +22,9 @@ def do_filter_search(self):
         self.do_post({
             "op": "searchByFilter",
             "query": "query q($input: [SearchInput]) {\n  searchResult: search(input: $input) {\n    items\n    __typename\n  }\n}",
-            "variables": {"input":[{"filters": [{"property":"kind", "values":["Pod"]}]}]}
+            "variables": {"input":[{"filters": [{"property":"namespace", "values":["default"]}], "limit": 1000}]}
```

Comment:
@jlpadilla @SherinV - do we need to classify the filters , for example kind:pod may match a lot of resources vs namespace:default .

Reply:
Yes, but my motivation for this change is because we had to code a special case for `kind` to match case insensitive (`ILIKE`) to keep compatibility with V1.